common.js

```diff
@@ -36,6 +36,16 @@ var vcwg = {
       status: "CG-DRAFT",
       publisher: "Credentials Community Group"
     },
+    "LD-PROOFS": {
+      title: "Linked Data Proofs",
+      href: "https://w3c-ccg.github.io/ld-proofs/",
```

Comment:
This currently 404's. Maybe it's not published yet?

Reply:
Not published yet.